### ISSUE TITLE
[template] Failure to generate templates for a given announcer

### ISSUE THREAD

**Issue by dhinojosa** [jreleaser/jreleaser#1446]
### Task List

- [x] Steps to reproduce provided
- [x] Stacktrace (if present) provided
- [x] Example that reproduces the problem (link to git repository is ideal)
- [x] Full description of the issue provided (see below)

### Steps to Reproduce

1. Install the latest release of JReleaser
2. Create a Java Project
3. Create a folder called _src/jreleaser/templates_
4. Open a terminal
5. `cd` into your project
6. Run `jreleaser template generate --announcer linkedin` or `jreleaser template generate --announcer twitter`, or any other announcer

### Expected Behaviour

I suspect an announcer should be created in the _src/jreleaser/templates_ folder

### Actual Behaviour

```sh
$  jreleaser template generate --announcer linkedin                                                                  danno@MacBook-Pro
[INFO]  Creating output directory /Users/danno/Development/mine/jreleaser-demos/jreleaser-demo-yml/src/jreleaser/templates
Unexpected error reading template for announcers/linkedin.tpl from classpath
(base) -----------------------------------------------------------------------
```

### Environment Information

- **Operating System**: MacOSX Ventura 13.4.1
- **JReleaser Version:** 1.7.0
- **JDK Version:** 20.0.1

Please paste stacktraces from `[out|build|target]/jreleaser/trace.log`.

```
[DEBUG] resolving template from classpath for init/jreleaser.yml.tpl
[INFO]  Writing file /Users/danno/Development/mine/jreleaser-demos/jreleaser-demo-yml/jreleaser.yml
[INFO]  JReleaser initialized at /Users/danno/Development/mine/jreleaser-demos/jreleaser-demo-yml
```

**Comment by aalmiray** [comment]
There's a subset of announcers that do not provide template generation support. The current lost of supported announcers is located at https://github.com/jreleaser/jreleaser/tree/f9d4315b81c0de1b188519be5c0b27b507f53e47/core/jreleaser-templates/src/main/resources/META-INF/jreleaser/templates/announcers

There are two options to solve this issue:

1. provide template support for missing announcers, using the basic message supported by each announcers
2. for those announcers where a generic message is not advisable a proper error message (instead of an exception) should be the recommended outcome. 

How to identify announcers of the second type: they do not provide a default value for their message/content/input properties.

**Comment by aalmiray** [comment]
🎉 This issue has been resolved in `v1.8.0` ([Release Notes](https://github.com/jreleaser/jreleaser/releases/tag/v1.8.0))
